packages/protocol/script/layer1/mainnet/DeployMainnetPacayaL1.s.sol

```diff
@@ -165,6 +164,7 @@ contract DeployMainnetPacayaL1 is DeployCapability {
         Ownable2StepUpgradeable(sharedResolver).transferOwnership(contractOwner);
         Ownable2StepUpgradeable(rollupResolver).transferOwnership(contractOwner);
         Ownable2StepUpgradeable(proofVerifier).transferOwnership(contractOwner);
+        Ownable2StepUpgradeable(store).transferOwnership(contractOwner);
```

Comment:
this will only transfer the ownership, the the new owner has to accept the ownership.

Reply:
I think the few lines starting at 170 should be commented out, also please also log which contracts' ownership should be accepted by admin.taiko.eth.

Reply:
Yes, those are only for reminding.